dask/dataframe/io/tests/test_parquet.py

```diff
@@ -134,3 +134,41 @@ def test_categorical():
         ddf2.loc[:1000].compute()
         df.index.name = 'index'  # defaults to 'index' in this case
         assert assert_eq(df, ddf2)
+
+
+@pytest.mark.parametrize('df,kwargs', [
+    (pd.DataFrame({'x': [3, 2, 1]}), {}),
+    (pd.DataFrame({'x': ['c', 'a', 'b']}), {'object_encoding': 'utf8'}),
+    (pd.DataFrame({'x': ['cc', 'a', 'bbb']}), {'object_encoding': 'utf8'}),
+    (pd.DataFrame({'x': [b'a', b'b', b'c']}), {'object_encoding': 'bytes'}),
+    (pd.DataFrame({'x': pd.Categorical(['a', 'b', 'c'])}), {}),
```

Comment:
These are also objects requiring object_encoding. It occurs to me that for the specific case of category labels, we could enable 'infer' after all.